.gitignore

```diff
@@ -272,3 +272,4 @@ src/mod/applications/mod_http_cache/test/test_aws
 src/mod/applications/mod_http_cache/test/test_aws.log
```

Comment:
Please remove this and rebase

Reply:
if you want to add this in .gitignore, just do it straight using tag TEL-0000.  no need to do a PR.